index.html

```diff
@@ -0,0 +1,304 @@
+          <h1>About me</h1>
+          <p>
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p> 
+      
+          <button>Get My Resume</button>
+  
+        </div>
+      </section>
+      <section class="myskills">
+        <div class="skills">
+          <div class="languageHeading">
+            <img src="/images/kite.png" alt="" class="kite" />
+            <h1>Languages</h1>
+          </div>
+          <ul class="LanguageSkill">
+            <li>javascript</li>
+            <li>ruby</li>
+            <li>html</li>
+            <li>css</li>
+          </ul>
+          <div class="frameworkheading">
+            <img src="./images/Rectangle 56.png" alt="" />
+            <h1>frameworks</h1>
+          </div>
+          <ul class="frameworkSkill">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+            <li>RSPec</li>
+          </ul>
+          <div class="skillotherheading">
+            <img src="/images/circle.png" alt="" />
+            <h1>Skills</h1>
+          </div>
+          <ul class="skilOther">
+            <li>Codekit</li>
+            <li>github</li>
+            <li>Codepen</li>
+            <li>gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <section class="details">
+
+        <div class="contact-p">
+             <p>
+              I'm always interested in hearing about new projects, so if you'd like
+              to chat please get in touch.
+            </p>
+        </div>
+        <form action="https://formspree.io/f/xyyvjvdn" method="POST">
+          <div class="ContactForm">
+              <div class="inputText">
+                      <input type="text" name="name" placeholder="Full name"
+                       required="" maxlength="30" type="text" class="input_t">
+                       <div class="line-"></div>
+                </div>
+                
+    
+              <div class="inputText">
+                  <input
+                   type="email"name="email" required=""
+                    placeholder="Email address" class="input_t">
+                      
+               </div>
+               
+    
+              <div class="inputTextarea">
+                  <textarea
+                   id="comment"
+                   name="comment"
+                   cols="50" rows="10" maxlength="500" required=""
+                   placeholder="Enter text here"
+                   type="text" class="text-area"></textarea>
+                </div>
+                
+                <div class="contact-button">
+                      <input type="Submit" value="Get In Touch" class="btn-touch">
+                </div>
+            </div>
```

Comment:
- [ ] to match Figma design kindly increase the width of the textarea to be equal to the other inputs

Your Design          | Figma
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/199476015-de0373c1-da4b-44db-a7f3-8640cf953b0e.png) | ![image](https://user-images.githubusercontent.com/89282221/199475915-9d27cb7b-c6d8-48e8-938b-c864ed396e80.png)

Reply:
brother that is optional as you even mention but it can't be the reason to stop approving the project brother .
this is a matter of padding size of which a new coder like cant measure it with bare eyes.
This is not okay brother

Reply:
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Reply:
> Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Reply:
placeholder="Enter text here"  that is the name of my placeholder not  this test of which I don't know where you have got it .
This project was okay to the level of the required  as per question

Reply:
@Robison-faike123 you are not required to measure it with bare eyes, despite all of the sizes being included in the Figma design, I didn't request to be 100% correct with the sizes, I only asked to make all the inputs have the same width since you are required to follow the Figma design as part of the project, anyways At Microverse, you're being prepared for a typical work environment where you may be required to keep to a given design as perfectly as possible, if you need another opinion you can always fill this [form](https://airtable.com/shrQAqnBwek5a0O0s), Cheers and happy coding!